cmd/envtool/tests.go

```diff
@@ -97,6 +100,8 @@ func runGoTest(ctx context.Context, args []string, total int, times bool, logger
 
 	var done int
 	results := make(map[string]*testResult, 300)
+	contexts := make(map[string]context.Context)
+	spans := make(map[string]trace.Span)
```

Comment:
Let's add those fields to the `testResult` struct

Reply:
Done